cmake/gsLibrary.cmake

```diff
@@ -213,6 +213,15 @@ set_target_properties(${PROJECT_NAME} PROPERTIES
 
 endif(GISMO_BUILD_LIB)
 
+list (FIND GISMO_OPTIONAL "gsOpennurbs" _index)
+if (${_index} GREATER -1)
+  # find_package(opennurbs REQUIRED)
+  include(${CMAKE_SOURCE_DIR}/external/gsOpenNurbs.cmake)
+
+  # TODO Check found if not cmake error
+  target_link_libraries(${PROJECT_NAME} ${OPENNURBS_LIBRARY_PATH})
+endif()
+
```

Comment:
Is it possible package/pull upstream  the zlib library from https://github.com/madler/zlib
 (now in external/zlib).
opennurbs includes zlib -- but zlib is also included for instance in Debian.
Would be nice to update opennurbs to find zlib and remove it from it's sources